ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py

```diff
@@ -57,7 +57,7 @@
     required: False
     type: bool
     description:
-    - Check if the interfaces are offline before regenerating the WWNN (World Wide Node Name). Possible values:- 'true' 'false'.
+    - Check if the interfaces are offline before regenerating the WWNN. Possible values:- 'true' 'false'.
```

Comment:
When True/False choices are rendered, it says Yes or No. Please remove all instances of: "Possible values:- 'true' 'false'"

Reply:
although the rendering converts it to yes and no but as per ansible guidelines: true false can be provided in multiple forms 
like 0,1 or yes or no, OR
true or false. Since our nimble spec has specified true or false and customers are used to it . I think it makes sense to leave it as it is. anyway there is no harm.